tests/spec/w3c/linter-rules/privsec-section-spec.js

```diff
@@ -26,69 +28,89 @@ describe("W3C Linter Rule - 'privsec-section'", () => {
         "See the [Self-Review Questionnaire](https://w3ctag.github.io/security-questionnaire/).",
     });
   });
+
   it("finds privacy section", async () => {
     const elem = doc.createElement("h2");
     elem.innerHTML = "the privacy of things";
     doc.body.appendChild(elem);
     const results = await rule.lint(config, doc);
+
     expect(results.length).toEqual(0);
     elem.remove();
   });
+
   it("finds just security sections", async () => {
     const elem = doc.createElement("h2");
     elem.innerHTML = "security of things";
     doc.body.appendChild(elem);
     const results = await rule.lint(config, doc);
+
     expect(results.length).toEqual(0);
     elem.remove();
   });
+
   it("ignores only considerations sections", async () => {
     const elem = doc.createElement("h2");
     elem.innerHTML = "Considerations for other things";
     doc.body.appendChild(elem);
     const results = await rule.lint(config, doc);
+
     expect(results.length).toEqual(1);
     elem.remove();
   });
+
   it("finds privacy considerations sections", async () => {
     const elem = doc.createElement("h2");
     elem.innerHTML = "Considerations of privacy of things";
     doc.body.appendChild(elem);
     const results = await rule.lint(config, doc);
+
     expect(results.length).toEqual(0);
     elem.remove();
   });
+
   it("finds security considerations sections", async () => {
     const elem = doc.createElement("h2");
     elem.innerHTML = "Considerations of security of things";
     doc.body.appendChild(elem);
     const results = await rule.lint(config, doc);
+
     expect(results.length).toEqual(0);
     elem.remove();
   });
+
   it("finds privacy and security considerations sections, irrespective of order", async () => {
     const elem = doc.createElement("h2");
     elem.innerHTML = "Privacy and Security Considerations";
     doc.body.appendChild(elem);
+
     expect((await rule.lint(config, doc)).length).toEqual(0);
     elem.innerHTML = "Security and Privacy Considerations";
+
     expect((await rule.lint(config, doc)).length).toEqual(0);
     elem.innerHTML = "Considerations Security Privacy";
+
```

Comment:
The newline-before-expect rule also looks weird here... should we just exclude it?

Reply:
I kinda like that rule, tbh. What might be better is to revise the test so it makes more sense.  

Where I would like to get to is for Prettier and Eslint to agree, and so they don't trash each other's changes.

Reply:
Okay, so that would be a future work 😁